apiclient/src/qs_api/ws.rs

```diff
@@ -323,6 +323,7 @@ impl ApiClient {
                     // The connection was not established, wait and try again
                     Err(e) => {
                         log::error!("Error connecting to QS WebSocket: {}", e);
+                        panic!("Error connecting to QS WebSocket: {}", address)
```

Comment:
Why do we want to panic here?

Reply:
Ah, sorry. That was a leftover from testing. If the API client builds a wrong URL, then this loops indefinietely. We should probably fix this a little more thoroughly.